src/themes/app.mat.styl

```diff
@@ -102,4 +102,16 @@ body.desktop .desktop-margin
     clear both
     display block
     visibility hidden
-    height 0px
\ No newline at end of file
+    height 0px
+
+.slide-toggle-enter-active,
```

Comment:
This `app.mat.styl` file seems to become a collection of unscoped styles.

Maybe try to put the styles into more specific places, e.g. put the transition styles into a new file and import it where needed?
Same for the `.edit-box` class, there's no need to have it site-wide.

Reply:
Yeah, makes sense. I just moved slide-toggle & edit-box to their own files. Will leave hoverScale in app.mat for now though.